### PR TITLE
Fix tests.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | https://github.com/yiisoft/db/issues/108

- [x] Rename namespace.
- [x] Remove code duplicate test db and db-pgsql.
- [x] Move tests db --> db-pgsql.
- [x] Run tests from db-pgsql.
- [x] Fix badge README.md.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,11 +36,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\Db\\Pgsql\\": "src"
-        }
-    },
-    "autoload-dev": {
```
Comment (xepozz) [inline composer.json]: Why did you delete this?

File: tests/ColumnSchemaBuilderTest.php [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 declare(strict_types=1);
 
-namespace yiiunit\framework\db\pgsql;
+namespace Yiisoft\Db\Pgsql\Tests;
 
 use Yiisoft\Db\Schemas\ColumnSchemaBuilder;
 use Yiisoft\Db\Tests\ColumnSchemaBuilderTest as AbstractColumnSchemaBuilderTest;
 
-final class ColumnSchemaBuilderTest extends AbstractColumnSchemaBuilderTest
```
Comment (xepozz) [inline tests/ColumnSchemaBuilderTest.php]: Why not `final`?

File: composer.json (hunk 1 above)
Comment (terabytesoftw) [reply composer.json]: Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue.

File: tests/ColumnSchemaBuilderTest.php (hunk 2 above)
Comment (terabytesoftw) [reply tests/ColumnSchemaBuilderTest.php]: Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue.

File: composer.json (hunk 1 above)
Comment (xepozz) [reply composer.json]: It's wrong, because if `db-mysql`'s tests are not valid that it means that `db-mysql` package implemented wrong, not `db` implemented.
Deleting this section will negatively affect to the autoloader.

Read the purpose https://getcomposer.org/doc/04-schema.md#autoload-dev

Comment (terabytesoftw) [reply composer.json]: The tests are not bad please check, remember db is the abstraction and db-mysql is the implementation, we currently have the code of both tests repeated, which is difficult to maintain, the idea is to have the tests in place, and unstoppable Each package, remember this package is large and must be done in parts.

Comment (terabytesoftw) [reply composer.json]: To explain it in db we will have the abstract classes of the tests, in each implementation the tests of each controller, then I will execute from db, for now the tests of the implementation that will verify all possible cases without duplicate code.

File: composer.json [hunk 3]
```diff
@@ -36,11 +36,7 @@
...
-        "psr-4": {
+            "Yiisoft\\Db\\Pgsql\\": "src",
             "Yiisoft\\Db\\Pgsql\\Tests\\": "tests"
```
Comment (samdark) [inline composer.json]: That does not look correct. There is no need to load tests when package is used in the application.

Comment (terabytesoftw) [reply composer.json]: I know what autoload and autoload-dev means, currently db loads the tests in autoload, I am investing it to be able to make everything independent and in the end place all the tests in autoload-dev with final class .
